File: 02_ProgrammingFundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_02_Exercise/_04_PrintAndSum.java
package _02_ProgrammingFundamentals._01_BasicSyntaxConditionalStatementsAndLoops._02_Exercise;

import java.util.Scanner;

public class _04_PrintAndSum {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        int nStart = Integer.parseInt(scan.nextLine());
        int nEnd = Integer.parseInt(scan.nextLine());

        // Add variable
        int sum = 0;
        
        // Print the numbers
        for (int i = nStart; i <= nEnd; i++) {
            System.out.print(i + " ");
            sum += i;
        }
        
        // Print result
        System.out.println();
        System.out.printf("Sum: %d", sum);
        
        scan.close();
    }
}

File: 24_C#Fundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_02_Exercise/_04_PrintAndSum/_04_PrintAndSum.cs
using System;

public class _04_PrintAndSum
{
    public static void Main(string[] args)
    {
        // Read input values
        int nStart = int.Parse(Console.ReadLine());
        int nEnd = int.Parse(Console.ReadLine());

        // Add variable
        int sum = 0;

        // Print numbers
        for (int i = nStart; i <= nEnd; i++)
        {
            Console.Write(i + " ");
            sum += i;
        }

        // Print result
        Console.WriteLine();
        Console.WriteLine($"Sum: {sum}");
    }
}
